### ISSUE TITLE
Windows Hello support

### ISSUE THREAD

**Issue by MarkusJx** [buttercup/buttercup-desktop#1089]
Hi,
I thought it would be nice if we could unlock a vault using Biometrics/a PIN with Windows Hello.
Buttercup doesn't already have such functionality, right?

**Comment by Claywd** [comment]
It did for a brief moment. I think they disabled it because of issues.

**Comment by perry-mitchell** [comment]
Would happily accept a PR for this. I won't have time to add it myself.

**Comment by MarkusJx** [comment]
Yeah, sure, I could work on this. I already created a node addon which adds windows hello support, but I'd have to see how to properly encrypt data with that as windows hello only adds support for signing data, so I'd have to use the credential vault... we'll see.
But I'll come back if I got something on that.
And if anyone has suggestions, please let me know.

**Comment by perry-mitchell** [comment]
@MarkusJx Please let me know if you need anything from me regarding how Buttercup works.. if I may assist at all. :pray:

**Comment by Claywd** [comment]
curious what the status "On Hold" means on this and what happened?

**Comment by EvilVir** [comment]
Hi, any progress on this feature? :)
